packages/one-app-runner/src/startAppContainer.js

```diff
@@ -0,0 +1,210 @@
+const { createHash } = require('node:crypto');
+
+const Docker = require('dockerode');
+
+const spawn = require('./asyncSpawn');
+const spawnAndPipe = require('./spawnAndPipe');
+
+const docker = new Docker();
```

Comment:
I'm not a fan of this being a file global...
maybe a separate file for sharing between startAppContainer.js & startApp.js? 🤔

Reply:
(this creates a socket/network connection to the docker daemon)